web-platform/website/src/components/Highlighter.tsx

```diff
@@ -33,6 +33,11 @@ const styles = {
         height: "30%",
         width: "100%",
       },
+      ".highlight-launch": {
```

Comment:
This class should be defined at the caller side and not included here. That is the whole point of letting classes be injected here.

Reply:
Right! Will update. Thanks.